docs/modules/operation/pages/deep-dive/performance-data-collection/resource-types.adoc

```diff
@@ -34,32 +34,53 @@ Here is the diskIOIndex resource type definition from `$OPENNMS_HOME/etc/datacol
 ----
 
 == Persistence selector strategies
-.Persistence Selector Strategies
-[options="header, autowidth"]
+
+.Persistence selector strategies
+[options="header" "autowidth", cols="1,2"]
 |===
-| Class                                                              | Description
-| org.opennms.netmgt.collection.support.PersistAllSelectorStrategy   | Persist all indexes
-| org.opennms.netmgt.collection.support.PersistRegexSelectorStrategy | Persist indexes based on JEXL evaluation.
+| Class
+| Description
+
+| `org.opennms.netmgt.collection.support.PersistAllSelectorStrategy`
+| Persist all indexes.
+
+| `org.opennms.netmgt.collection.support.PersistRegexSelectorStrategy`
+| Persist indexes based on JEXL evaluation.
 |===
 
 === PersistRegexSelectorStrategy
 
 The PersistRegexSelectorStrategy class takes a single parameter, `match-expression`, which defines a JEXL expression.
 On evaluation, this expression should return either true (persist index to storage) or false (discard data).
 
-
 == Storage strategies
-.Storage Strategies
-[options="header, autowidth"]
+
+.Storage strategies
+[options="header" "autowidth", cols="1,2"]
 |===
-| Class                                                           | Storage Path Value
-| org.opennms.netmgt.collection.support.IndexStorageStrategy      | Index
-| org.opennms.netmgt.collection.support.JexlIndexStorageStrategy  | Value after JexlExpression evaluation
-| org.opennms.netmgt.collection.support.ObjectNameStorageStrategy | Value after JexlExpression evaluation
-| org.opennms.netmgt.dao.support.FrameRelayStorageStrategy        | interface label + '.' + dlci
-| org.opennms.netmgt.dao.support.HostFileSystemStorageStrategy    | Uses the value from the hrStorageDescr column in the hrStorageTable, cleaned up for UNIX file systems.
-| org.opennms.netmgt.dao.support.SiblingColumnStorageStrategy     | Uses the value from an SNMP lookup of OID in sibling-column-name parameter, cleaned up for UNIX file systems.
-| org.opennms.protocols.xml.collector.XmlStorageStrategy          | Index, but cleaned up for UNIX file systems.
+| Class
+| Storage Path Value
+
+| `org.opennms.netmgt.collection.support.IndexStorageStrategy`
+| Index
+
+| `org.opennms.netmgt.collection.support.JexlIndexStorageStrategy`
+| Value after JexlExpression evaluation.
+
+| `org.opennms.netmgt.collection.support.ObjectNameStorageStrategy`
+| Value after JexlExpression evaluation.
+
+| `org.opennms.netmgt.dao.support.FrameRelayStorageStrategy`
+| interface label + '.' + dlci
+
+| `org.opennms.netmgt.dao.support.HostFileSystemStorageStrategy`
+| Uses the value from the hrStorageDescr column in the hrStorageTable, cleaned up for UNIX file systems.
+
+| `org.opennms.netmgt.dao.support.SiblingColumnStorageStrategy`
+| Uses the value from an SNMP lookup of OID in sibling-column-name parameter, cleaned up for UNIX file systems.
+
+| `org.opennms.protocols.xml.collector.XmlStorageStrategy`
+| Index, but cleaned up for UNIX file systems.
```

Comment:
Let's make it "cleaned up to make a valid directory name on supported file systems". These paths should work on more than just UNIX file systems. (repeat repeat comment)

Reply:
```suggestion
| Index, cleaned to make a valid directory name on supported file systems.
```